### PR TITLE
Cancel Case

### PR DESCRIPTION
**Pull request by murilovmachado**
Jira: https://bugs.benetech.org/browse/CHI-146

This PR:
- Creates the Cancel Case Flow
- Update Styles to match the mockups

Backend: https://github.com/tech-matters/hrm/pull/41

WIP:
- tests

Screen Shot 2020-07-17 at 13 59 30
WIP:
- tests

Screen Shot 2020-07-17 at 13 59 15

### CODE REVIEW

File: plugin-hrm-form/src/components/case/Case.jsx [hunk 1]
```diff
@@ -1,74 +1,140 @@
...
-  const counselor = props.counselorsHash[twilioWorkerId];
-  const date = new Date(createdAt).toLocaleDateString(navigator.language);
+  handleCancelNewCaseAndClose = async () => {
+    const { task, form } = this.props;
+    const { connectedCase } = form.metadata;
+    await cancelCase(connectedCase.id);
 
-  const saveAndEnd = async () => {
-    const { task, form } = props;
+    this.props.changeRoute('tabbed-forms', task.taskSid);
```
Comment (nickhurlburt) [inline plugin-hrm-form/src/components/case/Case.jsx:55]: When do we want to try react-router again?  (If at all.)

Comment (murilovmachado) [reply plugin-hrm-form/src/components/case/Case.jsx:55]: I prefer to try `react-router` again after all case management features get done.

Comment (nickhurlburt) [reply plugin-hrm-form/src/components/case/Case.jsx:55]: Sounds good to me.

File: plugin-hrm-form/src/components/case/Case.jsx [hunk 2]
```diff
@@ -1,74 +1,140 @@
...
+    isMenuOpen: false,
+    mockedMessage: null,
+  };
+
+  toggleCaseMenu = e => {
+    e.persist();
+    this.setState(prevState => ({ anchorEl: e.currentTarget || e.target, isMenuOpen: !prevState.isMenuOpen }));
+  };
+
+  handleMockedMessage = () => this.setState({ mockedMessage: <Template code="NotImplemented" />, isMenuOpen: false });
```
[the remark this answers is not in the document]
Comment (murilovmachado) [reply plugin-hrm-form/src/components/case/Case.jsx:46]: Yeah, I think it's a good idea to try reusing this.
We should definitely do something like this soon.

**Comment by murilovmachado** [comment]
deployed to staging.

**Pull request merged**